File: prime-numbers/prime.py
value = int(input("Enter the value..."))
count = 0
print("The prime numbers till ", value ," are ")
for i in range(2,value):
	for j in range(2,value):
		if(i%j == 0 and i!=j):
			count = count + 1
	if count < 1:
		print(i)
	count = 0

File: prime-numbers/prime.cpp
#include<iostream>
int main()
{
	int value, count = 0;
	std::cout<<"Enter a value...";
	std::cin>>value;
	std::cout<<"The prime numbers till "<<value<<" are"<<endl;
	for(int i = 2 ; i < value ; i++)
	{
		for (int j = 2; j < value; j++)
		{
			if(i % j == 0 && i != j)
				count++;
		}
		if(count<1)
			std::cout<<i<<"\n;
		count = 0;
	}
}